code-challenges.test.js

```
@@ -11,38 +14,207 @@
 // Run the file with the following command: $ yarn jest
 
 
-// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+// --------------------
+
+// 1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+
+
+// program to generate fibonacci series up to n terms
+
+// take input from the user
+
+// ReferenceError: number is not defined
+
+const fibonacci = (numer) =>{
+    for (let i = 2; i <= number; i++) {
+    n1 = n2;
+    n2 = nextTerm;
+    nextTerm = n1 + n2;
+   return (n1);
+}
+}
+console.log(fibonacci(6));
+console.log(fibonacci(10));
+
 
 // a) Create a test with expect statements for each of the variables provided.
 
```

Great job. This is the first part of the Red-green refactor process
1: Write the test first! 

2: Run your testing suite to see a failing test
Then as long as you have added your dependencies through the terminal command `yarn add jest`, you can run the test suite with `yarn jest`

And see that you have 1 test suite, 1 test that failed. The test should fail on the function name that is not defined because you have not created a function yet.